elm.json

```diff
@@ -14,4 +14,4 @@
     "test-dependencies": {
         "elm-explorations/test": "1.0.0 <= v < 2.0.0"
     }
-}
\ No newline at end of file
+}
```

Comment:
I cannot undo this newline here. Perhaps this is the time to learn how can be a commit reverted? :scream:

Reply:
Maybe it's because of editor settings. I wouldn't worry about it

Reply:
that's fine. your editor should not allow you to do that. I'm not sure why elm produces json without trailing line breaks...